pa_matchbatch.sh

```diff
@@ -22,7 +22,17 @@ overridesFile="/tmp/version_overrides.txt"
 OVERRIDE_PATH=$(realpath ${OVERRIDE_PATH:-${overridesFile}})
 
 # read in the auth token
-AUTH_TOKEN_ARG="-a ${JIRA_AUTH_TOKEN}"
+if [[ -z "$JIRA_AUTH_TOKEN" ]]; then
+    >&2 echo <<EOF
+Error: The JIRA_AUTH_TOKEN environment variable must be set.
+
+Create a token at https://id.atlassian.com/manage-profile/security/api-tokens and run
+export JIRA_API_TOKEN=$(echo -n '<username>@perforce.com:<token>' | base64 --wrap=0)
```

Comment:
The `base64` that comes with macos doesn't have a `wrap` setting, but I believe the using `break` is equivalent.

Reply:
Added a conditional based on `$OSTYPE`, though not sure it works as expected on macOS? Could you test?

Reply:
I think it should be
```
 if [[ "$OSTYPE" == "darwin"* ]]; then
```

Changing it to that works for me.

Reply:
@tvpartytonight I updated it